src/Hazelcast.Net.Testing/Remote/Resources/hazelcast.xml

```diff
@@ -58,5 +58,5 @@
     <capacity>10</capacity>
     <time-to-live-seconds>180</time-to-live-seconds>
   </ringbuffer>
-
+  <jet enabled="true"/>
```

Comment:
this is going to enable JET for all our tests by default, which is a good thing - but then there's a special other config file with JET enabled in the tests project that we don't need anymore... think you can find it and remove it?

Reply:
Done.